src/vm/eventpipebuffermanager.cpp

```diff
@@ -820,39 +833,23 @@ void EventPipeBufferManager::DeAllocateBuffers()
         pElem = m_pPerThreadBufferList->FindAndRemove(pElem);
         _ASSERTE(pElem != NULL);
 
-        SListElem<EventPipeBufferList*> *pCurElem = pElem;
+        SListElem<EventPipeBufferList *> *pCurElem = pElem;
         pElem = m_pPerThreadBufferList->GetNext(pElem);
-        delete(pCurElem);
+        delete (pCurElem);
 
         // Now that all the list elements have been freed, free the list itself.
-        delete(pBufferList);
+        delete (pBufferList);
         pBufferList = NULL;
     }
 }
 
-void EventPipeBufferManager::ResumeWriteEvent()
```

Comment:
Farewell ResumeWriteEvent. You were a good method but I am glad to see you go : )